src/Symfony/Component/Serializer/Context/Normalizer/DateTimeNormalizerContextBuilder.php

```diff
@@ -69,4 +69,13 @@ public function withCast(?string $cast): static
     {
         return $this->with(DateTimeNormalizer::CAST_KEY, $cast);
     }
+
+    /**
+     * Configures if the timezone should be used from the context and ignore any timezone
+     * defined in date strings.
+     */
+    public function withForceTimezone(?bool $forceTimezone): static
```

Comment:
idea: add a $force argument to withTimezone instead?

Reply:
Fixed :)

Reply:
Ait, updated :)